Examples/Io/Performance/ActsExamples/Io/Performance/SeedingPerformanceWriter.cpp

```diff
@@ -100,8 +97,9 @@ ActsExamples::ProcessCode ActsExamples::SeedingPerformanceWriter::writeT(
     nSeeds += regionVec.size();
     for (size_t i = 0; i < regionVec.size(); i++) {
       const Acts::Seed<SimSpacePoint>* seed = &regionVec[i];
-      ProtoTrack ptrack{seed->sp()[0]->index(), seed->sp()[1]->index(),
```

Comment:
Could you remove the line 92 above? The `HitParticlesMap` are retrieved twice from the eventStore.

Reply:
Fixed.